json_serializable/README.md

```diff
@@ -26,6 +26,8 @@ Given a library `example.dart` with an `Person` class annotated with
 [`JsonSerializable`]:
 
 ```dart
+import 'dart:convert';
```

Comment:
please don't add things to the default example!

Reply:
Reverted the changes to the example, and added an integration test which should help to show the use-case.